rubocop-capybara.gemspec

```
@@ -32,5 +32,5 @@ Gem::Specification.new do |spec|
     'rubygems_mfa_required' => 'true'
   }
 
-  spec.add_runtime_dependency 'rubocop', '~> 1.41'
+  spec.add_runtime_dependency 'rubocop', '~> 1.50'
```

Currently, `NewCops: disable` is set, so the following specification may be sufficient.

```suggestion
  spec.add_runtime_dependency 'rubocop', '>= 1.41', '< 2.0'
```

---

FYI: https://github.com/rubocop/rubocop-rspec/pull/1635#discussion_r1179187224

---

Sorry, I had overlooked that the patch version was not specified 😅.